src/libcadet/model/MultiChannelTransportModel.cpp

```diff
@@ -408,12 +409,26 @@ bool MultiChannelTransportModel::configureModelDiscretization(IParameterProvider
 			paramProvider.popScope();
 	}
 
+	
+	clearExchangeModels();
+
+	_exchange.push_back(nullptr);
+
+	if (paramProvider.exists("EXCHANGE_MODEL"))
+		_exchange[0] = helper.createExchangeModel(paramProvider.getString("EXCHANGE_MODEL"));
+
+	if (!_exchange[0])
+		_exchange[0] = helper.createExchangeModel("LINEAR_EX");
```

Comment:
I'd prefer to simply call this "LINEAR", similar to what we do for the binding configuration, but thats up to you